utils/objectUtils.js

```diff
@@ -0,0 +1,8 @@
+const has = (obj, path) => {
+  // Regex explained: https://regexr.com/58j0k
```

Comment:
Please add a small explanation along the regexpr link

Reply:
Do you mean something like: "You can have a analysis of the regexp and play with it on this RegExp playground link"?

Or something like: "This regexp will separe the string using '[', ']' and '.' as delimiter"

Reply:
The second one, then mention the first one

Reply:
The idea is that in the current form I have to enter the link to get the idea, but maybe should be optional to dig into the intention

Reply:
Yeah sounds comfortable for developers to have a little description so can avoid to enter in the link. Will add it. Thanks!

Reply:
added on
https://github.com/victor-heliomar/i18n-translation-generator/commit/3fc14587f191735e711cff336e541b9416b6d9db

let me know what do you think

Reply:
A lot better! Great work!